civihr_employee_portal/templates/civihr-employee-portal-custom-reports.tpl.php

```diff
@@ -105,6 +105,9 @@ class="btn btn-lg btn-secondary-outline text-uppercase">
             </div>
         </footer>
     </section>
+    <div data-graph-section="summary">
```

Comment:
I haven't looked into the specific of this table, but do you think it would be possible to have something in the html file like (just a very rough draft):
```html
<table data-summary-table-tpl>
    <thead>
        <tr>
            <th>Column 1</th>
            <th>Column 2</th>
            <th>Column 3</th>
            <th>Column 4</th>
        </tr>
    </thead>
    <tbody>
        <!-- tr to clone and fill with data -->
        <tr>
            <td>{column1Value}</td>
            <td>{column2Value}</td>
            <td>{column3Value}</td>
            <td>{column4Value}</td>
        </tr>
    </tbody>
</table>
```
which then you could get with javascript, clone, do the placeholder replacement with actual value and then put in the template?